bigtable/admin.go

```diff
@@ -1663,3 +1664,53 @@ func (ac *AdminClient) UpdateBackup(ctx context.Context, cluster, backup string,
 	_, err = ac.tClient.UpdateBackup(ctx, req)
 	return err
 }
+
+// GetIamPolicy gets the IAM access control policy for the specified backup.
+func (ac *AdminClient) GetIamPolicy(ctx context.Context, cluster, backup string) (*iampb.Policy, error) {
+	ctx = mergeOutgoingMetadata(ctx, ac.md)
+	prefix := ac.instancePrefix()
+	backupPath := prefix + "/clusters/" + cluster + "/backups/" + backup
+
+	req := iampb.GetIamPolicyRequest{Resource: backupPath}
+	policy, err := ac.tClient.GetIamPolicy(ctx, &req)
+	if err != nil {
+		return nil, err
+	}
+	return policy, nil
+}
+
+// SetIamPolicy sets the IAM access control policy for the specified backup.
+// Replaces any existing policy.
+func (ac *AdminClient) SetIamPolicy(ctx context.Context, cluster, backup string, policy *iampb.Policy) (*iampb.Policy, error) {
```

Comment:
I think `SetIamPolicy` and `TestIamPolicy` can be deleted once `BackupIAM` above returns `*iam.Handle`.